proc.c

```diff
@@ -439,13 +439,13 @@ scheduler(void)
         switch (pr->state)
         {
           case RUNNING:
-            pr->rutime++;
+            pr->rutime += (delta_ticks / QUANTA);
```

Comment:
shouldn't it just be rutime += delta_ticks? this way, we're counting how many quantas it waits/runs/sleeps for instead of actual ticks. if a full quanta isn't used [due to yield or sleep], then it would be as if it never happened. no stats would get recorded